resources/languages.ts

```diff
@@ -0,0 +1,7 @@
+export const Languages = ['en', 'de', 'fr', 'ja', 'cn', 'ko'] as const;
+
+export type Lang = typeof Languages[number];
+
+export type NonEnLang = Exclude<Lang, 'en'>;
+
+export const isLang = (lang?: string): lang is Lang => Languages.includes(lang as Lang);
```

Comment:
```suggestion
export const isLang = (lang?: string): lang is Lang => Languages.includes(lang);
```

I think this is unnecessary.  `Languages` is a `string[]` and `lang` is a `string`.

Reply:
Also I thought I typed this in but maybe I edited it away accidentally.  Can `Languages` be `languages` instead? I know cactbot is not consistent with capitalization, but ideally variables should be `camelCase`, and I'd like to get there eventually.

Reply:
I wasn't sure if exported variables should be capitalized like classes, or if they should be camelCase?

Reply:
Renamed, made the adjustment as well, and tested to make sure it works since I had some weird issues with a slightly different approach that was along the same vein.